src/jit/codegenarm64.cpp

```diff
@@ -5114,779 +5114,6 @@ void CodeGen::genStoreLclTypeSIMD12(GenTree* treeNode)
 
 #endif // FEATURE_SIMD
 
-#ifdef FEATURE_HW_INTRINSICS
-#include "hwintrinsic.h"
-
-instruction CodeGen::getOpForHWIntrinsic(GenTreeHWIntrinsic* node, var_types instrType)
```

Comment:
This logic was removed and replaced with new logic in `hwintrinsiccodegenarm64.cpp`, to mirror the `xarch` file.